pkg/services/accesscontrol/ossaccesscontrol/builtin_roles.go

```diff
@@ -0,0 +1,44 @@
+package ossaccesscontrol
+
+import (
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+var builtInRoles = []accesscontrol.RoleDTO{
```

Comment:
Why not use a map here to make o(1) complex the getBuildInRole function?

Reply:
Yes, makes sense, this code just moved from the previous version, so some optimization may be applied.